api/src/flows.ts

```diff
@@ -421,9 +421,11 @@ class FlowManager {
 
 		const handler = this.operations.get(operation.type)!;
 
-		const options = applyOptionsData(operation.options, keyedData);
+		let options = null;
 
 		try {
+			options = applyOptionsData(operation.options, keyedData);
```

Comment:
If we proceed with the defaulting to options, this change could be added to batch.
```suggestion
			options = applyOptionsData(options, keyedData);
```